index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <title>Hello Microverse</title>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href="style.css" rel="stylesheet">
+</head>
+
+<body> 
+    <h1>Hello Microverse</h1>
+</body>
+
+</html>
```

Comment:
[OPTIONAL]: 
⚠️ This is your Hello Microverse project, and this is your third re-review. For that reason, I will be approving this PR. You did a great job adding the linters to the project. However, they are not running, because you merged the previous pull request, and closed the PR.  This line would be giving a linter error ❌ because we're missing a newline at the end of the file.

💡 This will be VERY important for all future projects. I can let it go, only for Hello Microverse:

* Never close or merge a PR before you get an approval to your project from a code reviewer. If you accidentally close a PR that was reviewed, you can reopen it as described on this link: https://gist.github.com/Farhan-Haseeb/fd4cbb476de4e2931c322d7bacb75e5f#:~:text=You%20need%20the%20rights%20to,checkbox%20and%20mark%20as%20Open.
This will ensure that all your linters actually run (and you get a green check ✅ when you pass them). 
Because it's Hello Microverse, this can be approved with this linter issue. But it's a reason for invalidating your review for future projects.

Linters are here to help make all your projects look professional to a potential future employer's eyes! They are a friend you must embrace. ⭐

Reply:
Hello @melaniesigrid . I accidentally closed the first PR. I didn't know how to open it. This reference will be important for future use. Thnak you